### PR TITLE
Musical Concert Project

### PR DESCRIPTION
**Pull request by abiola-arobieke**
In this milestone:

The home and about page was created using HTML and CSS.
Both pages were made to be responsive on different screens.
The UI design was parsed
Data was fetched into the index page dynamically using Javascript.
The "more" and "hide" button on the index page for the mobile screen was made to toggle, show, and hide data fetched.
Index page was designed to fetched data based on adjustment of the browser or screen size

Here is the link to the loom video for the documentation:

https://www.loom.com/share/0dfe7296e65049779d284bc773f79712

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,241 +16,120 @@
...
-- **[key_feature_3]**
+- **Fetch Data Dynamically**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
```
Comment (Ismailco) [inline README.md:87]: - [ ] Kindly make sure to create a presentation video about your project and add the link to the PR description. here is the requirement of the video from the project requirements:
Screenshot_20230325_163956

File: README.md [hunk 2]
```diff
@@ -23,241 +16,120 @@
...
-
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://abiola-arobieke.github.io/musical-concert/)
```
Comment (Ismailco) [inline README.md]: - [ ] Good job using Github pages. however, the link is re-directing to a `404` page, kindly make sure that the link is working properly and linked to a live version of the project

Screenshot_20230325_164419

File: index.html [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+            </div>
+            <div class="footer-content align-self-center">
+                <h6>2015 Creative Commons Korea. Some Rights Reserved.</h6>
+                <p>
+                    달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                    자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.
+                </p>   
+            </div>
+        </div>
+    </footer>
```
Comment (Ismailco) [inline index.html]: - [ ] The Footer and Speakers section should not be displayed in the mobile view on the Home page, kindly make sure to remove them from the Mobile view so you're website looks identical to the provided design

|Your website|Provided design|
|-|-|
|Screenshot_20230325_163752|Screenshot_20230325_163807|

File: about.html [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+          <h6>2015 Creative Commons Korea. Some Rights Reserved.</h6>
+          <p>
+            달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈
+            저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다. 자세한
+            이용기능 내용과 허락조건은
+            http://creativecommons.org/licences/by/2.0/kr 를 보세요.
+          </p>
+        </div>
+      </div>
+    </footer>
```
Comment (Ismailco) [inline about.html]: - [ ] Kindly make sure that there is a `partner` section on the `About` page, in the Mobile view so your website looks identical to the provided design

|You're website|Provided design|
|-|-|
|Screenshot_20230325_163703|Screenshot_20230325_163718|

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @abiola-arobieke, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Great Work 💯
✔️ Nice Design
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @abiola-arobieke, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**